documentation/components/tutorial-flow-grid.asciidoc

```diff
@@ -598,6 +598,7 @@ grid.addColumn(TemplateRenderer
 * Every property in the template needs to be defined in the `withProperty()` method.
 * `\[[item.name]]` is Polymer syntax for binding properties for a list of items.
 See the https://polymer-library.polymer-project.org/3.0/api/elements/dom-repeat[Polymer 3 documentation] for more.
+* When using a custom web-component or a Vaadin element in a template renderer, remember to import the component. This can be done using link:https://vaadin.com/docs/v14/flow/polymer-templates/tutorial-template-subtemplate.html[`@JsModule`]  or link:https://vaadin.com/api/platform/com/vaadin/flow/component/dependency/Uses.html[`@Uses`], if the component has a server-side counterpart. This will ensure all StyleSheet, HtmlImport, JavaScript dependencies for component are loaded when the Grid is used.
```

Comment:
I think it's good to add a "the" before component here:

This will ensure all StyleSheet, HtmlImport, JavaScript dependencies for component are loaded when the Grid is used.

Reply:
Done